### PR TITLE
desc internal addresses with native VPs

### PR DESCRIPTION
**Pull request by tzemanovic**
related to #47, #283 and #38

### CODE REVIEW

File: tech-specs/src/explore/design/ledger/accounts.md [hunk 1]
```diff
@@ -35,6 +38,10 @@ As implied by their name, accounts for implicit addresses exist as a possibility
 
 Established addresses are created by a ledger transaction, which can create any number of new account addresses. The users are not in control of choosing the address as it's derived from the current address nonce, which is changed after every newly established address.
 
+#### Internal transparent addresses
+
+There will be a static set of internal addresses that integrate certain functionality into the ledger via a dedicated module, such as proof-of-stake module and IBC module. The internal accounts use [native validity predicates](vp.md#native-vps) to validate transactions that interact with their module. A native module will use the [dynamic storage sub-space](#dynamic-storage-sub-space) to store all the data relevant to their functionality (e.g. PoS parameters, bond pool, IBC state and proofs).
```
Comment (cwgoes) [inline tech-specs/src/explore/design/ledger/accounts.md:43]:
```suggestion
There will be a static set of internal addresses that integrate certain functionality into the ledger via a dedicated module, such as the proof-of-stake module and the IBC module. The internal accounts use [native validity predicates](vp.md#native-vps) to validate transactions that interact with their module. A native module will use the [dynamic storage sub-space](#dynamic-storage-sub-space) to store all the data relevant to their functionality (e.g. PoS parameters, bond pool, IBC state and proofs).
```

File: tech-specs/src/explore/design/ledger/vp.md [hunk 2]
```diff
@@ -8,7 +8,28 @@ Each [account](accounts.md) is associated with exactly one validity predicate (V
...
+    // Set of all the addresses whose VP was triggered by the transaction
+    verifiers: HashSet<Address>,
+) -> bool;
+```
+
+The host functions available to call from inside the VP code can be found in [docs generated from code](https://anomanetwork.github.io/anoma/rustdoc/anoma_vm_env/imports/vp/index.html#functions).
+
+## Native VPs
+
+The native VPs follow the same interface as WASM VPs and rules for how they are [triggered by a transaction](tx.md#tx-execution). They can also call the same host functions as those provided in [WASM VPs environment](wasm-vm.md#vps-environment) and must also account any computation for gas usage.
```
Comment (cwgoes) [inline tech-specs/src/explore/design/ledger/vp.md:32]: How do they account for gas usage? Are there functions they can call?

**Review by cwgoes — APPROVED** [review]
ACK & thanks! See minor question.

File: tech-specs/src/explore/design/ledger/vp.md (hunk 2 above)
Comment (tzemanovic) [reply tech-specs/src/explore/design/ledger/vp.md:32]: Yup, we have dedicated functions in the VM `host_env` module which we can parametrize to use outside wasm too.

**Review by yito88 — APPROVED** [review]
LGTM

**Pull request merged**